src/SWP/Bundle/TemplateEngineBundle/Model/Container.php

```diff
@@ -77,7 +76,7 @@ class Container implements ContainerInterface, TenantAwareInterface, Timestampab
     protected $widgets;
 
     /**
-     * @var TenantInterface
+     * @var string
```

Comment:
This TemplateEngineBundle supports only Doctrine ORM entiries. To solve it properly, a Container class should be created in `WebRendererBundle` (I will rename it in next PR to `CoreBundle`) which extends `TemplateEngineBundle/Model/Container.php` and in this new class we should add the `TenantAwareInterface`, override the mapping file for Container class to use tenant code. This way we will decouple MultiTenancy from `TemplateEngineBundle`.